hr_family/models/res_partner.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, models, fields
+from openerp.exceptions import Warning as UserError
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    # added to port old field fam_spouse_employer on hr_employee
+    # from previous version
+    employer = fields.Char("Employer")
```

Comment:
I'm not so sure about this. If somebody really wants to save stuff here, she'd add a company partner I guess, this would also then contain the fields to record what is relevant about the employer. I'd either not touch the employer field at all or make it a m2o to res.partner too

Reply:
I thought about doing that. I would have to search for the name and assign the many2one.  if no match is found, then create another new partner . I will do that now. It is good for the OCA to have a moduel that preserves data.

Reply:
3722b8005ad3a28827b89dbe0563bdeb77be68fa